hw14task1.py

```diff
@@ -0,0 +1,52 @@
+"""
+Homework.
+"""
+# Files.
+# Напишите программу, которая создает текстовый
+# файл(если его нету) "students.txt". Запишите в файл
+# список студентов, номер группы, их оценки. Каждый
+# студент на новой строке. Откройте файл и прочитайте всю
+# информацию из него. Напечатайте общее количество студентов,
+# количество студентов для каждой группы и среднюю оценку для
+# каждой группы. Допишите эту информацию в конец файла.
+# Передусмотрите возможные ошибки и обработайте их.
+
+
+def process_students_data():
+    try:
+        with open("students.txt", "w", encoding="utf-8") as file:
```

Comment:
do not hardcode filename and pass it to the function

Reply:
Corrected